content/documentation/pages/5-feature-guides/1-streams/7-java-dsl.md

```diff
@@ -4,4 +4,255 @@ title: 'Java DSL'
 description: 'How to create streams using Java'
 ---
 
-Introduction to the Java DSL
+# Stream Java DSL
+
+Instead of using the shell to create and deploy streams, you can use the Java-based DSL provided by the `spring-cloud-dataflow-rest-client` module.
+The Java DSL is a convenient wrapper around the `DataFlowTemplate` class that enables creating and deploying streams programmatically.
+
+To get started, you need to add the following dependency to your project, as follows:
+
+```xml
+<dependency>
+	<groupId>org.springframework.cloud</groupId>
+	<artifactId>spring-cloud-dataflow-rest-client</artifactId>
+	<version>{project-version}</version>
+</dependency>
+```
+
+<!--NOTE-->
+
+A complete sample can be found in the [Spring Cloud Data Flow Samples Repository](https://github.com/spring-cloud/spring-cloud-dataflow-samples).
```

Comment:
It'd be nice to link to that sample directly as opposed to the root folder. See: https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/htmlsingle/#_java_dsl